src/pages/Explorer/Explorer/explorer.module.scss

```diff
@@ -0,0 +1,242 @@
+@import '~@santiment-network/ui/mixins';
+
+.buttonsContainer {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+}
+
+.buttons {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+}
+
+.topicon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    font-family: 'Proxima Nova';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;    
+    color: #7A859E;
+
+    padding: 12px 16px 12px 12px;
+    height: 40px;
+    border: 1px solid var(--mystic);
+    box-sizing: border-box;
+    border-radius: 100px;
+    
+    &:not(:last-child) {
+        margin-right: 8px;
+    }
+
+    &:hover, &.active {
+        border: 1px solid var(--jungle-green);
+        background: #EDF8F5;
+        color: var(--jungle-green);
+
+        svg {
+            fill: var(--jungle-green);
+        }
+    }
+
+    svg {
+       fill: var(--waterloo);
+    }
+}
+
+.header {
+    width: 100%;
+    padding: 17px 24px;
+    border: 1px solid var(--porcelain);
+    border-top-left-radius: 8px;
+    border-top-right-radius: 8px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    h3 {
+        font-family: 'Proxima Nova';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 18px;
+        line-height: 26px;
+        color: var(--rhino);
```

Comment:
Use `san-webkit`'s typography classes [`body-1`](https://github.com/santiment/san-webkit/blob/master/src/styles/text.css#L40) and color class [`c-black`](https://github.com/santiment/san-webkit/blob/master/src/styles/text.css#L73)